include/fastrtps/utils/collections/ResourceLimitedVector.hpp

```diff
@@ -440,6 +440,11 @@ class ResourceLimitedVector
         collection_.pop_back();
     }
 
+    value_type* data()
+    {
+        return collection_.data();
+    }
+
```

Comment:
Could you also add the `const` version of this?

Reply:
Done